src/components/Redoc/Redoc.tsx

```diff
@@ -57,7 +57,22 @@ export class Redoc extends React.Component<RedocProps> {
           <OptionsProvider value={options}>
             <GlobalCss />
             <StyledHeader>
-              <UnifiedNav position="relative" property={{ name: 'DOCS', searchParams: [] }} />
+              <UnifiedNav
+                position="relative"
+                property={{ name: 'DOCS', searchParams: [] }}
+                // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                // @ts-ignore
+                darkMode={false}
```

Comment:
Ah, seems like they have a separate public facing [interface](https://github.com/10gen/consistent-nav/blob/42f55f655e37ec1e957ea61bd4c1c0f54c73b42a/src/index.tsx#L34) for `UnifiedNav` over `NavBar`. We can probably create follow-up PRs to fix, but I'm okay with these `ts-ignore` for now